src/js/utils/html-utils.js

```diff
@@ -6,44 +6,64 @@
  */
 (function(define) {
     'use strict';
-    define(['underscore', 'edx-ui-toolkit/js/utils/string-utils'], function(_, StringUtils) {
-        var escape, interpolateHtml, HTML, template;
+    define(['underscore', 'jquery', 'edx-ui-toolkit/js/utils/string-utils'], function(_, $, StringUtils) {
+        var ensureHtml, interpolateHtml, joinHtml, HTML, template, setHtml, append, prepend;
 
         /**
-         * Helper function to return a string flagged as an HTML snippet.
+         * Creates an HTML snippet.
+         *
+         * The intention of an HTML snippet is to capture a string that
```

Comment:
The intention of an HTML snippet is to communicate that the string it represents contains HTML that has been safely escaped as necessary.